### PR TITLE
Add static review-summary with badges

### LINKED ISSUE #112: Update Reviews Page with Summary and Labels
# Documents and Resources

- [ ] [Full Mockup](https://projects.invisionapp.com/share/4Z12PZBKD6WR#/screens/467140917)

# Mockup

Image

### PR DESCRIPTION
**Pull request by will-byrne**
Closes #112

### CODE REVIEW

**Review by scottaubrey — APPROVED** [review]

File: src/reviews/index.scss [hunk 1]
```diff
@@ -103,3 +103,37 @@ h1.article-review-status__heading {
...
+  @include buttons.action-button();
+  @include typography.summary-badge();
+  margin-bottom: 6px;
+  margin-right: 6px;
+  padding: 5px 10px 6px 10px;
+}
+
+.summary-badge-list__item--active {
+  color: white;
+  background-color: settings.$color-badge-active;
```
Comment (discodavey) [inline src/reviews/index.scss:138]: Needs to be in alphabetical order

File: src/reviews/reviews.ts [hunk 2]
```diff
@@ -29,6 +29,42 @@ const wrapWithHtml = (reviews: string, doi: string, noHeader: boolean): string =
...
+              This is a <span class="summary-text__highlight">landmark</span> paper and a <span class="summary-text__highlight">tour-de-force</span> that ties together decades of advances in electron microscopy
+              to produce a dataset of both breadth and extreme technical quality whose very existence will have profound
+              and lasting influence on neuroscience. The manuscript is extensive and well-illustrated, and the data,
+              methods and analyses are made available to the community in an exemplary manner. The work represents
+              ambitious, large-scale biological resource generation at its apotheosis.
+            </p>
+
+            <h3 class="summary-heading">Importance of claim</h3>
+            <ol class="summary-badge-list">
+              <li class="summary-badge-list__item summary-badge-list__item--active">Landmark</li>
```
Comment (discodavey) [inline src/reviews/reviews.ts:44]: Do we need a cursor on these as it does not do anything?

File: src/sass/_typography.scss [hunk 3]
```diff
@@ -106,16 +106,24 @@
...
 }
 
-@mixin article-review-timeline-text() {
-  color: settings.$color-text;
+@mixin summary-heading() {
+  color: settings.$color-text-inactive;
   font-family: settings.$font-family-primary;
   font-size: 14px;
-  line-height: 32px;
+  line-height: 30px;
```
Comment (discodavey) [inline src/sass/_typography.scss:120]: @chris-huggins There are various line-heights for 14px font size. Is this correct as we 24px, 26px, 30px and 32px for 14px font size

File: src/reviews/index.scss (hunk 1 above)
Comment (scottaubrey) [reply src/reviews/index.scss:138]: Quick PR done for this [here](https://github.com/elifesciences/enhanced-preprints/pull/173) - would you mind giving it an approval

**Pull request merged**